files/image_config/procdockerstats/procdockerstats

```diff
@@ -0,0 +1,212 @@
+        elif unit.lower() == UNITS_KB.lower():
+            value_converted = value_to_convert * 1000
+            return int(round(value_converted))
+        elif unit.lower() == UNITS_MB.lower():
+            value_converted = value_to_convert * 1000 * 1000
+            return int(round(value_converted))
+        elif unit.lower() == UNITS_MiB.lower():
+            value_converted = value_to_convert * 1024 * 1024
+            return int(round(value_converted))
+        elif unit.lower() == UNITS_GiB.lower():
+            value_converted = value_to_convert * 1024 * 1024 * 1024
+            return int(round(value_converted))
+
+    def create_docker_dict(self, dict_list):
+        dockerdict = {}
+        for row in dict_list[0:]:
+            cid = row.get('CONTAINER ID')
+            if cid:
+                key = 'DOCKER_STATS|' + str(cid) 
+                dockerdict[key] = {}
+                dockerdict[key]['NAME'] = row.get('NAME')
+                
+                splitcol = row.get('CPU %')
+                cpu = re.split("%", str(splitcol))
+                dockerdict[key]['CPU%'] = str(cpu[0])
+
+                splitcol = row.get('MEM USAGE / LIMIT')
+                memuse = re.split(" / ", str(splitcol))
+                # converting MiB and GiB to bytes
+                dockerdict[key]['MEM'] = str(self.convert_to_bytes(memuse[0]))
+                dockerdict[key]['MEM_LIMIT'] = str(self.convert_to_bytes(memuse[1]))
+
+                splitcol = row.get('MEM %')
+                mem = re.split("%", str(splitcol))
+                dockerdict[key]['MEM%'] = str(mem[0])
+
+                splitcol = row.get('NET I/O')
+                netio = re.split(" / ", str(splitcol))
+                dockerdict[key]['NET_IN'] = str(self.convert_to_bytes(netio[0]))
+                dockerdict[key]['NET_OUT'] = str(self.convert_to_bytes(netio[1]))
+                
+                splitcol = row.get('BLOCK I/O')
+                blockio = re.split(" / ", str(splitcol))
+                dockerdict[key]['BLOCK_IN'] = str(self.convert_to_bytes(blockio[0]))
+                dockerdict[key]['BLOCK_OUT'] = str(self.convert_to_bytes(blockio[1]))
+
+                dockerdict[key]['PIDS'] = row.get('PIDS')
+        return dockerdict
+
+    def update_dockerstats_command(self):
+        cmd = "docker stats --no-stream -a"
+        data = self.run_command(cmd)
+        if not data:
+            log_error("'{}' returned null output".format(cmd))
+            return False
+        dockerdata = self.format_docker_cmd_output(data)
+        if not dockerdata:
+            log_error("formatting for docker output failed")
+            return False
+        # wipe out all data from state_db before updating
+        self.state_db.delete_all_by_pattern('STATE_DB', 'DOCKER_STATS|*')
+        for k1,v1 in dockerdata.iteritems():
+            for k2,v2 in v1.iteritems():
+                self.update_state_db(k1, k2, v2)
```

Comment:
In the two `if` statements above, you explicitly return False upon failures. I suggest you explicitly return True if we reach the end of the function (i.e., success) for consistency.